lighthouse-core/lib/stack-packs.js

```diff
@@ -47,6 +39,18 @@ const stackPacksToInclude = [
     packId: 'joomla',
     requiredStacks: ['js:joomla'],
   },
+  {
+    packId: 'next.js',
+    requiredStacks: ['js:next'],
+  },
+  {
```

Comment:
placing all non-CMS / framework type things last, so that more specific stack advice is shown first.